counterfit/core/frameworks.py

```diff
@@ -35,18 +13,18 @@ def run(self):
         raise NotImplementedError
 
     @abstractmethod
-    def pre_attack_processing(self, cfattack: CFAttack):
+    def pre_attack_processing(self, cfattack):
         pass
 
     @abstractmethod
-    def post_attack_processing(self, cfattack: CFAttack):
+    def post_attack_processing(self, cfattack):
         pass
```

Comment:
This function `post_attack_processing` was marked as `abstractmethod` but not enforcing the functionality of @abstractmethod. We need to either enforce all sub-classes to implement this function and this function should `raise NotImplementedError ` or completely remove this function from this class and implement this function in respective subclasses.